tasklist/client/e2e/pages/TaskDetailsPage.ts

```diff
@@ -73,6 +74,7 @@ class TaskDetailsPage {
     this.valueColumnHeader = this.variablesTable.getByRole('columnheader', {
       name: 'Value',
     });
+    this.taskCompletedMsg = page.getByText('Task completed');
```

Comment:
Do not use abbreviations
I also believe you can a better selector, try some `getByRole` please

```suggestion
    this.taskCompletionNotification = page.getByText('Task completed');
```